gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -464,6 +464,13 @@ public void createBucket(String bucketName, CreateBucketOptions options) throws
     } catch (InterruptedException e) {
       Thread.currentThread().interrupt();
       throw new IOException("Failed to create bucket", e);
+    } catch (IOException e) {
+      if (ApiErrorExtractor.INSTANCE.itemAlreadyExists(e)) {
+        throw (FileAlreadyExistsException)
```

Comment:
Unnecessary cast?

Reply:
No, it's necessary -`initCause` returns `Throwable`.